cmc-domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/ClaimTest.java

```diff
@@ -47,13 +45,13 @@ public void isNotEqualWhenDifferentClaimObjectGiven() {
 
     @Test
     public void isNotEqualWhenOnlyOneFieldIsDifferent() {
-        Claim claim1 = customCreatedAt(NOW_IN_LOCAL_ZONE.plusNanos(1));
-        Claim claim2 = customCreatedAt(NOW_IN_LOCAL_ZONE);
+        Claim claim1 = customCreatedAt(DatesProvider.NOW_IN_LOCAL_ZONE.plusNanos(1));
```

Comment:
Would be good to restore static imports.

Reply:
done. probably we need to update our checkStyle config to catch this for future.